File: eCommerce/ClientApp/src/app/shared/classes/products/productShort.ts
import { MarketStatus } from "../product";

export interface ProductShort {
 id?: number;
 title?: string;

 quantity?: number;
 price?: number;
 discountRate?: number;

 rating?: any;
 ratingVotes?: number;

 marketStatus?: MarketStatus;
 image?: string;
}

File: Entities/ViewModels/Products/ProductShort.cs
using Entities.Enums;

namespace Entities.ViewModels.Products
{
    public class ProductShort
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public int Quantity { get; set; }

        public double Price { get; set; }
        public double DiscountRate { get; set; }

        public double Rating { get; set; } = 0.0;
        public int RatingVotes { get; set; } = 0;
        public MarketStatus MarketStatus { get; set; }


        public string Image { get; set; }
    }
}
